react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -373,8 +435,10 @@ class ConnectionStatsTable extends Component {
                     { this._renderBitrate() }
                     { this._renderPacketLoss() }
                     { isRemoteVideo ? this._renderE2eRtt() : null }
+                    { isRemoteVideo ? this._renderRegion() : null }
```

Comment:
I recall you asking me about this. I think you can also do `isRemoteVideo && this._renderRegion()` and get the same result, but either words for me.